test/request_source/request_source_plugin_test.cc

```diff
@@ -0,0 +1,114 @@
+#include "envoy/common/exception.h"
+
+#include "external/envoy/source/common/config/utility.h"
+#include "external/envoy/test/mocks/stats/mocks.h"
+#include "external/envoy/test/test_common/file_system_for_test.h"
+#include "external/envoy/test/test_common/utility.h"
+
+#include "common/request_source_plugin_impl.h"
+
+#include "test/test_common/environment.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+using nighthawk::request_source::DummyPluginRequestSourceConfig;
+using nighthawk::request_source::FileBasedPluginRequestSourceConfig;
+using ::testing::Test;
+
+class DummyRequestSourcePluginTest : public Test {
+public:
+  DummyRequestSourcePluginTest() : api_(Envoy::Api::createApiForTest(stats_store_)) {}
+  Envoy::Api::ApiPtr api_;
+  Envoy::Stats::MockIsolatedStatsStore stats_store_;
+}; // RequestSourcePluginTest
```

Comment:
This comment seems unusual, this is what we usually do with namespaces. The rest of the code doesn't do this, so for consistency please consider removing it.

Reply:
Understood.